packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -134,7 +134,7 @@ describe('--schema from parent directory', () => {
 function cleanSnapshot(str: string): string {
   return str
     .replace(/\d+ms/g, 'XXms')
-    .replace(/\d+s/g, 'XXs')
+    .replace(/\d+s/g, 'XXms')
```

Comment:
The problem with this function is it's not even called in all but two tests. Other tests rely on the snapshot serializer replacing the milliseconds, so even with this change some tests will sometimes fail: https://github.com/prisma/prisma/issues/10896

Reply:
I closed the issue since my cleanup refactor PR adressed that here